packages/compartment-mapper/test/snapshots/test-policy.js.md

```diff
@@ -0,0 +1,113 @@
+# Snapshot report for `test/test-policy.js`
```

Comment:
Informative errors for policy problems are still possible despite it's being applied as early as building the compartment map